app/main.py

```diff
@@ -1,3 +1,67 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int or float) -> callable:
+        self.km = km
+
+    def __str__(self) -> callable:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> callable:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int or float) -> Distance(int or float):
```

Comment:
```suggestion
    def __add__(self, other: int or float) -> Distance(int or float):
```
use Union[Distance, int, float]